website/docs/addon_deadline_admin.md

```diff
@@ -16,107 +16,182 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 
 - Enable `Tools > Super User Mode` in Deadline Monitor
-
 - Go to `Tools > Configure Plugins...`, find `AYON` in the list on the left side, find location of AYON
 executable. It is recommended to use the `ayon_console` executable as it provides a bit more logging.
-
 - In case of multi OS farms, provide multiple locations, each Deadline Worker goes through the list and tries to find the first accessible
  location for itself.
-
 - provide your Ayon server address and API key for service account generated in step 6 of [Preparation](#Preparation)
+- provide additional servers if needed, they should follow this format: `{server url}@{token}`, e.g., `http://11.2.3.47@myapikey1`.
 
-![Configure plugin](assets/deadline_configure_plugin.png)
+![](assets/deadline/deadline_configure_plugin.png)
 
-### AYONTileAssembler Plugin
-To setup tile rendering copy the `OpenPypeTileAssembler` plugin to the repository;
-`[AYON]\ayon\modules\deadline\repository\custom\plugins\OpenPypeTileAssembler` > `[DeadlineRepository]\custom\plugins\OpenPypeTileAssembler`
+#### CelAction
 
-### Pools
+It's required for CelAction submissions via deadline addon.
+![](assets/deadline//celaction_dl_plugin.png)
 
-The main pools can be configured at `ayon+settings://deadline/publish/CollectDeadlinePools`, which is applied to the rendering jobs.
+#### HarmonyAYON
 
-The dependent publishing job's pool uses `ayon+settings://deadline/publish/ProcessSubmittedJobOnFarm`. If nothing is specified the pool will fallback to the primary pool above.
+It's required for Harmony submissions via deadline addon.
+![](assets/deadline/harmonyayon_dl_plugin.png)
 
-:::note maya tile rendering
-The logic for publishing job pool assignment applies to tiling jobs.
+#### UnrealEngine5
+
+It extends the official Unreal5 Plugin to support Unreal Perforce submissions.
+It implements `PreLoad` scripts that syncs your project to change list id within the exported `changelist` before rendering.
+For more info, check [Version Control - Support for Unreal and Deadline](addon_version_control_admin_get_started.md#support-for-unreal-and-deadline) and [Version Control - Deadline](addon_version_control_artist.md#deadline)
+
+:::info
+Unreal submissions via deadline addon requires `ayon-version-control` addon and configured (P4 credentials etc.). See [Version Control - Addon Settings](addon_version_control_admin_settings.md#addon-settings)
 :::
 
-### Authentication
-Deadline supports username and passwords authentication to protect webservice from unwanted requests. (In Deadline `Tools>Configure Repository Options>Web Service Settings>Require Authentication`)
+![](assets/deadline//unreal5_dl_plugin.png)
 
-For this admin needs to enable authentication requirement for DL in `ayon+settings://deadline/require_authentication`.
 
-Artists then need to provide their credentials in `Site Settings`.
+#### HuskStandaloneSubmitter
```

Comment:
```suggestion
#### Husk Standalone Deadline Plug-in
```

Reply:
Thank you https://github.com/ynput/ayon-documentation/pull/301/commits/fce37fa156ace9c67c0fb17c7b50bd41108d4a8f